lib/firefox.js

```diff
@@ -13,7 +13,7 @@ var GARBAGE = [
   /\[JavaScript Warning: "TypeError: "[\w\d]+" is read-only"\]/,
   /JavaScript strict warning: /,
   /\#\#\#\!\!\! \[Child\]\[DispatchAsyncMessage\]/,
-  /JavaScript strict warning: resource:\/\/\/modules\/sessionstore\/SessionStore\.jsm/,
+  /JavaScript strict warning: resource:\/\/\/modules\/sessionstore\/SessionStore\.jsm/, // eslint-disable-line max-len
```

Comment:
Can you concatenate Regular Expressions?
Do half these even need to be regular expressions vs plain strings?